tools/moth.py

```diff
@@ -328,15 +334,35 @@ def __init__(self, path, seed):
         self.path = path
         self.seed = seed
         self.pointvals = []
+        self.catmod = None
+
+        try:
+            catmod = SourceFileLoader(
+                'catmod',
+                os.path.join(path, 'category.py')).load_module()
+            assert all([
+                hasattr(catmod, 'make'),
+                hasattr(catmod, 'points'),
+                type(catmod.points) is list,
+            ])
+            self.catmod = catmod
+            self.pointvals.extend(catmod.points)
```

Comment:
As stated in the commit message, this follows the precedent set earlier -- if individual puzzles are dynamic, they don't all have to be dynamic; also, a puzzle may be both dynamic and static. Category follows this same pattern. Corollary: if it's not good for Category, Puzzle needs to change as well. :smile: